src/Tickets/Emails/Email/Ticket.php

```diff
@@ -134,38 +58,14 @@ public function get_title(): string {
 	}
 
 	/**
-	 * Get email heading.
+	 * Get default recipient.
 	 *
 	 * @since 5.5.9
 	 *
-	 * @return string The email heading.
+	 * @return string
 	 */
-	public function get_heading(): string {
-		$heading = tribe_get_option( static::$option_heading, $this->get_default_heading() );
-
-		/**
-		 * Allow filtering the email heading for Ticket.
-		 *
-		 * @since 5.5.9
-		 *
-		 * @param string          $heading  The email heading.
-		 * @param string          $id       The email id.
-		 * @param Tribe__Template $template Template object.
-		 */
-		$heading = apply_filters( 'tec_tickets_emails_ticket_heading', $heading, self::$id, $this->template );
-
-		/**
-		 * Allow filtering the email heading globally.
-		 *
-		 * @since TBD
-		 *
-		 * @param string          $heading  The email heading.
-		 * @param string          $id       The email id.
-		 * @param Tribe__Template $template Template object.
-		 */
-		$heading = apply_filters( 'tec_tickets_emails_heading', $heading, self::$id, $this->template );
-
-		return $this->format_string( $heading );
+	public function get_default_recipient(): string {
+		return '{attendee-email}';
```

Comment:
Again, not sure if this will work, as the ticket email is sent out to the purchaser, and attendee (if IAC). Also, we should use underscores for the placeholder format.

```suggestion
		return '{attendee_email}';
```

Reply:
Yeah. I figured we'd be changing this when we start implementing the data that goes into it. I just wanted to put in a placeholder for now.